static/docs/commands-reference/install.md

```diff
@@ -1,16 +1,43 @@
 # install
 
-Install dvc hooks into the repository
+Install DVC hooks into the Git repository to automate certain common actions
 
 ## Synopsis
 
 ```usage
     usage: dvc install [-h] [-q] [-v]
 ```
 
-## Installed hooks
-- pre-commit : dvc status
-- post-checkout : dvc checkout
+## Description
+
+As designed DVC combines an intelligent data repository with using a regular SCM
+like Git to store code and configuration files. With `dvc install` the two are
+more tightly integrated, causing certain actions to happen automatically. This
+saves the DVC user from having to remember to perform both DVC and SCM actions.
+
+There are two modes of tracking files in a DVC workspace:
+
+* Data files are managed by DVC in the DVC cache
+* Code, configuration and DVC files are managed by an SCM like Git
+
+**Checkout** For any given SCM branch or tag, the SCM checks-out the DVC files
+corresponding to that branch or tag. The DVC files in turn refer to data files
+in the DVC cache by checksum. Hence, switching from one SCM branch or tag to
+another, the SCM retrieves the corresponding DVC files. By default that leaves
+the workspace in a state where the DVC files refer to data files other than what
+is currently in the workspace. The user at this point should run `dvc checkout`
+so that the data files will match the current DVC files.
+
+**Commit** When committing a change to the SCM repository, that change possibly
+requires rerunning the pipeline to reproduce the workspace results. It is
+helpful to know automatically this as a reminder to run `dvc repro`.
+
+## Installed SCM hooks
+
+* Git `pre-commit` hook executes `dvc status` to inform the user about the
```

Comment:
does status hook rejects `git commit` if there are some uncommitted DVC files or repro needs to be run?

Reply:
No it doesn't

Reply:
That is - I ran a test modifying a file, and ran `git commit`.  The pre-commit hook simply gave `dvc status` and then I was able to commit the file to Git.

Hence there's nothing more to say in the documentation.